### ISSUE TITLE
Rvision stoped loading

### ISSUE THREAD

**Issue by Eli-S-Bridge** [swarm-lab/Rvision#40]
Hi. I have been using Rvison for a while now, but it recently quit working (see errors below). I updated cmake and installed the latest openCV with ROpenCVLite. Got the same errors. 

For what it's worth, there is a file called Rvision.so in:
/Library/Frameworks/R.framework/Versions/4.0/Resources/library/opencv/lib/libopencv_calib3d.4.3.dylib
  Referenced from: /Library/Frameworks/R.framework/Versions/4.0/Resources/library/Rvision/libs/

Thanks for any help....

> library(Rvision)
Error: package or namespace load failed for ‘Rvision’ in dyn.load(file, DLLpath = DLLpath, ...):
 unable to load shared object '/Library/Frameworks/R.framework/Versions/4.0/Resources/library/Rvision/libs/Rvision.so':
  dlopen(/Library/Frameworks/R.framework/Versions/4.0/Resources/library/Rvision/libs/Rvision.so, 6): Library not loaded: /Library/Frameworks/R.framework/Versions/4.0/Resources/library/opencv/lib/libopencv_calib3d.4.3.dylib
  Referenced from: /Library/Frameworks/R.framework/Versions/4.0/Resources/library/Rvision/libs/Rvision.so
  Reason: image not found

> library(ROpenCVLite)
> isOpenCVInstalled()
[1] TRUE
> opencvVersion()
[1] "Version 4.5.1"

**Comment by Eli-S-Bridge** [comment]
Manually deleted the Rvision library folder and reinstalled. Working now. Sorry for the trouble.

**Comment by sjmgarnier** [comment]
@Eli-S-Bridge Yes, this is normal. Every time a new version of OpenCV is compiled and installed, Rvision needs to be recompiled as well to be linked to it. I need to figure out a way to automatically detect this and throw a message to users when they need to reinstall Rvision. It's not on the top of my priority list at the moment (I'm currently focused on GPU support and video buffering), so I'll keep this thread open in case other people run into the same issue. Thanks.
